common/common.protocol.md

```diff
@@ -0,0 +1,25 @@
+# General Requirements
+
+## Exposure of Dataspace Protocol Versions
+
+Connectors implementing the Dataspace Protocol might operate on different versions. Therefore, it is necessary that they can discover the supported versions of each other in reliable and unambiguous manner. Each Connector must expose information of at least one Dataspace Protocol Version that it supports. The method expose this information is defined through the protocol bindings and might vary amongst them.
+
+In any way, a Connector must respond to a respective request by providing a JSON-LD object containing an array of supported versions with at least one item. The item connects the version tag (`version` attribute) with the absolute URL path segment of the root path for all endpoints of this version. The following example specifies that this Connector offers version `1.0` endpoints at `<host>/some/path/v1`.
+
+```
+{
+    "@context": "https://w3id.org/dspace/v0.8/context.json",
+    "protocolVersions": [
+       {
+            "version": "1.0",
+            "path": "/some/path/v1" 
+       }
+   ]
+}
+```
+
+This data object must comply to the [JSON Schema](schema/version-schema.json) and the [SHACL Shape](shape/version-shape.ttl). The value of the `version` attribute follows the [Semantic Versioning 2.0.0](https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string) and thereby has a numeric order on major, minor, and patch versions.
```

Comment:
@sebbader-sap we discussed in the meeting that we would leave out the semantic versioning here. This is to be decided as part of the DSP releasing process. The endpoint itself is not directly affected.

Reply:
I have rolled back the commit with the semver part. Please check it again.

Reply:
looks good to me